src/commands/stac-validate/README.md

```diff
@@ -0,0 +1,32 @@
+# stac-validate
+
+Validate STAC files
+
+## Usage
+
+stac-validate <options> [...location]
+
+### Arguments
+
+| Usage         | Description                            | Options |
+| ------------- | -------------------------------------- | ------- |
+| [...location] | Location of the STAC files to validate |         |
+
+### Options
+
+| Usage                  | Description                            | Options  |
+| ---------------------- | -------------------------------------- | -------- |
+| --config <str>         | Location of role configuration file    | optional |
```

Comment:
Aren't all options and flags optional?

Reply:
there are quite a few that are not optional?